tools/plugins/com.liferay.ide.project.ui/plugin.xml

```diff
@@ -66,6 +66,25 @@
                class="org.eclipse.core.resources.IResource">
          </selection>
       </wizard>
+      <wizard
+            category="com.liferay.ide.newWizards"
+            finalPerspective="com.liferay.ide.eclipse.ui.perspective.liferay"
+            icon="icons/e16/liferay_logo_16.png"
+            id="com.liferay.ide.project.ui.workspace.NewLiferayWorkspaceWizard"
+            name="Liferay Workspace"
+            preferredPerspectives="com.liferay.ide.eclipse.ui.perspective.liferay,org.eclipse.jst.j2ee.J2EEPerspective"
```

Comment:
The preferred perspective should be the new Liferay Workspace, not just the normal "liferay" perspective.